PathKitTests/ManipulationTests.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  ManipulationTests.swift
+//  PathKit
+//
+//  Created by Marius Rackwitz on 26.8.15.
+//  Copyright © 2015 Cocode. All rights reserved.
+//
+
+import Foundation
+import XCTest
+import PathKit
+import CatchingFire
+
+
+class ManipulationTests : XCTestCase {
+
+    var fixtures: Path {
+        return Path(__FILE__).parent() + "Fixtures"
+    }
+
+    var fixtureFile: Path {
+        return fixtures + "file"
+    }
+
+    var copiedFile: Path {
+        let path = Path("file")
+        if !path.exists {
+            try! fixtureFile.copy(path)
+        }
+        return path
+    }
+
+    override func setUp() {
+        super.setUp()
+        Path.current = try! Path.uniqueTemporary()
```

Comment:
Nope, because it's in the setUp. If that fails, not any test is failing, but it's an error with the test suite itself, so it's legit to do an assertion at this place.